models/User.php

```diff
@@ -460,7 +460,7 @@ public function touchLastSeen()
      */
     public function isOnline()
     {
-        return $this->getLastSeen() > $this->freshTimestamp()->subMinutes(5);
+        return $this->last_seen > $this->freshTimestamp()->subMinutes(5);
```

Comment:
@Samuell1 I don't mind using `getLastSeen` here - we just simply needed the `last_seen` timestamp to be touched after login.